.github/workflows/SpellCheck.yml

```diff
@@ -0,0 +1,15 @@
+name: Spell Check
+
+permissions: {}
+
+on: [pull_request]
+
+jobs:
+  typos-check:
+    name: Spell Check with Typos
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Actions Repository
+        uses: actions/checkout@v4
```

Comment:
```suggestion
        uses: actions/checkout@v4
        with:
          persist-credentials: false
```

Reply:
Jeepers; how does actions/checkout@v4 even get any credentials when there is a toplevel `permissions: {}`?

Reply:
IIRC, if you pass an empty dict, it populates it with some default values.

Reply:
I might be misremembering.

Reply:
Just to be safe, let's be explicit:

```yaml
permissions:
  contents: read
```